conda/recipes/libcudf/recipe.yaml

```
@@ -0,0 +1,233 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+# Copyright (c) 2018-2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  minor_version: ${{ (version | split('.'))[:2] | join('.') }}
+  cuda_version: ${{ (env.get('RAPIDS_CUDA_VERSION') | split('.'))[:2] | join(".") }}
+  cuda_major: ${{ (env.get('RAPIDS_CUDA_VERSION') | split('.'))[0] }}
+  date_string: ${{ env.get("RAPIDS_DATE_STRING") }}
+  head_rev: ${{ git.head_rev(".")[:8] }}
+
+recipe:
+  name: libcudf-split
+
+cache:
+  source:
+    path: ../../..
+
+  requirements:
+    build:
+      - ${{ compiler('c') }}
+      - ${{ compiler('cxx') }}
+      - ${{ compiler('cuda') }} =${{ cuda_version }}
```

Let `cuda-version` constrain the compiler version. We verified that this is sufficient for both CUDA 11 and CUDA 12 in the RMM PR.
```suggestion
      - ${{ compiler('cuda') }}
```